modules/rosa-deleting-aws-resources-cli.adoc

```diff
@@ -0,0 +1,100 @@
+====
+
+.Prerequisites
+
+* You have deleted your ROSA cluster. For more information, see the _Deleting a cluster_ section.
+
+.Procedure
+
+. Delete the OpenID Connect (OIDC) provider that you created for Operator authentication in your cluster:
++
+[source,terminal]
+----
+$ aws iam delete-open-id-connect-provider --open-id-connect-provider-arn <oidc_provider_arn> <1>
+----
+<1> Replace `<oidc_provider_arn>` with the Amazon Resource Name (ARN) of the OpenID Connect (OIDC) resource that you created to authenticate the cluster Operators. You can list the OIDC providers in your account by running `$ aws iam list-open-id-connect-providers`.
+
+. Delete the cluster-specific Operator IAM roles:
++
+[IMPORTANT]
+====
+You must delete the cluster before you remove the Operator roles.
+====
++
+.. List the policies that are attached to one of the cluster-specific IAM roles:
++
+[source,terminal]
+----
+$ aws iam list-attached-role-policies --role-name <operator_role_name> <1>
+----
+<1> Replace `<operator_role_name>` with the role name of a cluster-specific Operator role that you created for the cluster. Specify the role name and not the full ARN. You can list the roles in your account by running `$ aws iam list-roles`.
++
+[TIP]
+====
+The IAM role and policy ARNs include the role prefix that is specified when the STS resources are created. The default prefix is `ManagedOpenShift`.
+====
++
+.. Detach the policy from role:
++
+[source,terminal]
+----
+$ aws iam detach-role-policy --role-name <operator_role_name> --policy-arn <account_wide_policy_arn> <1>
+----
+<1> Replace `<account_wide_policy_arn>` with the ARN of the attached policy.
++
+.. Delete the role:
++
+[source,terminal]
+----
+$ aws iam delete-role --role-name <operator_role_name>
+----
++
+.. Repeat the steps to delete each of the cluster-specific Operator roles for the cluster.
+
+. Delete the Operator policies that you created for the cluster. The following command deletes a single policy:
++
+[source,terminal]
+----
+$ aws iam delete-policy --policy-arn <operator_policy_arn>  <1>
+----
+<1> Replace `<operator_policy_arn>` with the ARN of an Operator policy that you created for the cluster. You can list the policies in your account by running `$ aws iam list-policies`.
++
+Repeat this step to delete each of the Operator policies for the cluster.
+
+. Delete the account-wide IAM roles and policies that you created for this cluster. 
+.. List the inline policy for one of the account-wide IAM roles:
++
+[source,terminal]
+----
+$ aws iam list-role-policies --role-name <account_wide_role_name> <1>
+----
+<1> Replace `<account_wide_role_name>` with the role name of an account-wide IAM role that you created for the cluster. Specify the role name and not the full ARN. You can list the roles in your account by running `$ aws iam list-roles`.
```

Comment:
These policies and roles are not "created for the cluster", they are account-wide and can be reused across multiple clusters of the same OpenShift minor.

Reply:
This is now resolved.